firebaseai/FirebaseAIExample.xcodeproj/project.pbxproj

```diff
@@ -614,8 +618,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/firebase/firebase-ios-sdk.git";
 			requirement = {
-				kind = upToNextMajorVersion;
-				minimumVersion = 11.13.0;
+				kind = revision;
+				revision = e3110da07b49112d154f68c3df2cf22e2227233c;
```

Comment:
Reminder: We need to switch this to
```
kind = upToNextMajorVersion;
minimumVersion = 12.0.0;
```
after the next release / before merging.

Reply:
Done in #1730. Please merge to this branch.

Reply:
Thanks Paul!